source/features/list-prs-for-file.tsx

```diff
@@ -23,32 +23,33 @@ function getHovercardUrl(prNumber: number): string {
 	return buildRepoURL('pull', prNumber, 'hovercard');
 }
 
-const buttonId = 'rgh-list-prs-for-file-button';
-const popoverId = 'rgh-list-prs-for-file-popover';
+const buttonId = 'rgh-list-prs-for-file-';
+let count = 0;
```

Comment:
There are multiple items on the page, the ID needs to change. I noticed that the previous PR only worked after a reload